content/Lab2/30_Cleanup/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Cleanup"
+weight: 30
+---
+
+[Setup Cleanup - Destroy created resources](../30_Cleanup/1_cleanup.md)
```

Comment:
what does setup cleanup mean? Can we rename this to Cleanup - Destroy created resources and rename the respective md titles so that it matches with this text e.g. 1_network_resources.md title should be AWS Network Resources - Description